src/routes/queues.ts

```diff
@@ -3,40 +3,35 @@ import { RequestHandler, Request } from 'express'
 import { Job } from 'bull'
 import { Job as JobMq } from 'bullmq'
 
-import { BullBoardQueues, BullBoardQueue } from '../@types'
-
-interface ValidMetrics {
-  total_system_memory?: string
-  redis_version?: string
-  used_memory?: string
-  mem_fragmentation_ratio?: string
-  connected_clients?: string
-  blocked_clients?: string
-}
+import * as api from '../@types/api'
+import * as app from '../@types/app'
+// REVIEW: this is enough of an argument to move these constants to the top level, WDYT?
```

Comment:
Sounds reasonable

As long as it doesn't introduce more layers in the compiled dist

Reply:
WDYM? Is that something I have to worry about?
If compiled size is a focus point, it would be great to add something like [bundlesize](https://github.com/siddharthkp/bundlesize) to the CI tests so we can track it properly in PRs :)~

Reply:
I mean that if a TS project tries to resolve files that are outside of its root, it may add additional levels of folders in the compiled dist. Thus it may have an impact on paths within the dist (since we build both a client app and a server app and they rely on each others positions within dist) :)

Reply:
Gotcha! I'll make sure to check that